impeller/aiks/aiks_unittests.cc

```diff
@@ -174,18 +175,23 @@ bool GenerateMipmap(const std::shared_ptr<Context>& context,
     return false;
   }
   pass->GenerateMipmap(std::move(texture), std::move(label));
+  if (async_submit) {
+    return buffer->SubmitCommandsAsync(pass, context->GetResourceAllocator());
```

Comment:
Just based off of reading this, maybe it should be `EncodeAndSubmitCommandsAsync`.

Reply:
Yeah that is a good suggestion, i will do that (not today, but when I update this tomrrow) 😄

Reply:
Done